core/src/components/split-pane/split-pane.tsx

```diff
@@ -69,6 +69,24 @@ export class SplitPane implements ComponentInterface {
     }
   }
 
+  componentWillLoad() {
+    if (this.contentId === undefined) {
+      console.warn(`[DEPRECATED][ion-split-pane] Using the [main] attribute is deprecated, please use the "contentId" property instead:
+BEFORE:
+  <ion-split-pane>
+    ...
+    <div main>...</div>
+  </ion-split-pane>
+
+AFTER:
+  <ion-split-pane contentId="main-content">
```

Comment:
```suggestion
  <ion-split-pane content-id="main-content">
```
same as above

Reply:
Not really... in angular it would be contentId, not content-id